package.json

```diff
@@ -1,16 +1,23 @@
 {
   "name": "npmgraph",
   "version": "2.7.1",
+  "type": "module",
   "author": {
     "name": "Robert Kieffer",
     "url": "http://github.com/broofa",
     "email": "robert@broofa.com"
   },
   "license": "MIT",
+  "browserslist": [
+    "since 2017-06"
+  ],
   "dependencies": {
-    "@hpcc-js/wasm": "0.3.14",
+    "@hpcc-js/wasm": "1.4.1",
+    "d3": "6.6.2",
+    "d3-graphviz": "4.0.0",
     "http-errors": "1.8.0",
-    "preact": "10.5.7"
+    "react": "17.0.2",
```

Comment:
Why the switch?

Reply:
I was only using `preact` because I was using `htm`.  I was only using `htm` because I didn't want a build step.  Now, with Parcel, I'd rather stick with the recommended modules unless/until there's a need

Reply:
I say this because React is still larger than Preact, so if there’s no advantage in using React directly I think it’s best to stick to Preact

Reply:
Let's revisit this after I get this PR landed and we have a better feel for what the production build size looks like.